stacks/fdo/docker-stack.yml

```diff
@@ -65,13 +64,42 @@ services:
       log4j_configuration_file: log4j2.xml
     volumes:
       - owner-target:/home/fdo/target
-    networks:
-      - fdonetwork
+    network_mode: host
+
+  pri-fdo-manufacturer:
+    image: andresportainer/pri-fdo-manufacturer:latest
+    expose:
+      - "8039"
+    ports:
+      - "8039:8039"
+    environment:
+      manufacturer_keystore_password: RabSIZz43lympm8z
+      manufacturer_api_password: 05EV9CbHbAQANc1t
+      manufacturer_ssl_keystore-password: Sw2cqdcc1IM
```

Comment:
Should these be hard coded or variables?

Reply:
We agreed it was fine that way for the PoC, at least for now.